File: app/src/main/java/org/hydev/themeapplytools/activity/MainActivity.java
package org.hydev.themeapplytools.activity;

import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import org.hydev.themeapplytools.databinding.ActivityMainBinding;
import org.hydev.themeapplytools.utils.ThemeShareDialogUtils;
import org.hydev.themeapplytools.utils.ThemeUtils;

public class MainActivity extends AppCompatActivity {
    private static final String GITHUB_URL = "https://github.com/VergeDX/ThemeApplyTools";
    private static final String ME_COOLAPK_URL = "https://coolapk.com/u/506843";

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        ActivityMainBinding activityMainBinding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(activityMainBinding.getRoot());

        // Open dark mode will reCreate activity.
        ThemeUtils.darkMode(this);

        activityMainBinding.mcvApply.setOnClickListener(l -> {
            Intent intent = new Intent(MainActivity.this, ApplyThemeActivity.class);
            startActivity(intent);
        });

        activityMainBinding.mcvGetDirectLink.setOnClickListener(l -> {
            Intent intent = new Intent(MainActivity.this, GetDirectLinkActivity.class);
            startActivity(intent);
        });

        activityMainBinding.mcvLearnHow.setOnClickListener(l -> {
            Intent intent = new Intent(MainActivity.this, LearnHowActivity.class);
            startActivity(intent);
        });

        activityMainBinding.mcvExploreGithub.setOnClickListener(l -> ThemeShareDialogUtils.openBrowser(this, GITHUB_URL));
        activityMainBinding.mcvMeInCoolapk.setOnClickListener(l -> ThemeShareDialogUtils.openBrowser(this, ME_COOLAPK_URL));
    }
}

File: app/src/main/java/org/hydev/themeapplytools/activity/MainActivity.kt
package org.hydev.themeapplytools.activity

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import org.hydev.themeapplytools.databinding.ActivityMainBinding
import org.hydev.themeapplytools.utils.ThemeShareDialogUtils.openBrowser
import org.hydev.themeapplytools.utils.ThemeUtils

class MainActivity : AppCompatActivity() {
    companion object {
        private const val GITHUB_REPO_URL = "https://github.com/VergeDX/ThemeApplyTools"
        private const val COOLAPK_ME_HOMEPAGE = "https://coolapk.com/u/506843"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val activityMainBinding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(activityMainBinding.root)

        // Open dark mode will reCreate activity.
        ThemeUtils.darkMode(this)

        activityMainBinding.mcvApply.setOnClickListener { startActivity(Intent(this, ApplyThemeActivity::class.java)) }
        activityMainBinding.mcvGetDirectLink.setOnClickListener { startActivity(Intent(this, GetDirectLinkActivity::class.java)) }
        activityMainBinding.mcvLearnHow.setOnClickListener { startActivity(Intent(this, LearnHowActivity::class.java)) }
        activityMainBinding.mcvExploreGithub.setOnClickListener { openBrowser(this, GITHUB_REPO_URL) }
        activityMainBinding.mcvMeInCoolapk.setOnClickListener { openBrowser(this, COOLAPK_ME_HOMEPAGE) }
    }
}
